optuna/testing/samplers.py

```diff
@@ -76,3 +76,9 @@ def sample_independent(
         return super(FirstTrialOnlyRandomSampler, self).sample_independent(
             study, trial, param_name, param_distribution
         )
+
+
+class FixedSeedRandomSampler(optuna.samplers.RandomSampler):
+    def __init__(self) -> None:
+        seed = 42
```

Comment:
We may want to avoid hard-coding a magic number in the logic. How about extracting this as an argument or somewhere else?